cdap-notifications/src/main/java/co/cask/cdap/notifications/service/kafka/KafkaNotificationService.java

```diff
@@ -79,8 +81,9 @@ public KafkaNotificationService(CConfiguration cConf, KafkaClient kafkaClient, D
     this.feedManager = feedManager;
     this.ack = KafkaPublisher.Ack.LEADER_RECEIVED;
     this.nbPartitions = cConf.getInt("kafka.num.partitions");
-
+    this.notificationTopic = cConf.get(Constants.Notification.KAFKA_TOPIC_PREFIX);
```

Comment:
Thanks @anew for the comments. Kafka topics allow ASCII alphanumerics, '.', '_' and '-', as shown [here](https://github.com/apache/kafka/blob/trunk/core/src/main/scala/kafka/common/Topic.scala). If we validate Kafka topics, then our string validation will depend on Kafka's implementation. Instead, can we depend on Kafka to validate strings? Please let me know your thoughts.